packages/enzyme-test-suite/package.json

```diff
@@ -35,8 +35,8 @@
     "jsdom": "^6.5.1",
     "lodash.isequal": "^4.5.0",
     "mocha-wrap": "^2.1.2",
-    "object.assign": "^4.1.0",
     "object-inspect": "^1.6.0",
+    "object.assign": "^4.1.0",
```

Comment:
not sure why this got re-sorted, what version of npm did you use to install it?

Reply:
It looks like I accidentally add new package with yarn. will fix this.